### ISSUE TITLE
Possible bug in the "Train a seq2seq model on the..." example

### ISSUE THREAD

**Issue by ptresende** [facebookresearch/ParlAI#279]
When running:
```python examples/train_model.py -m seq2seq -t babi:task1k:1 -bs 8 -e 1 -mf /tmp/model_s2s```

I'm getting the following exception:
```
Traceback (most recent call last):
  File "examples/train_model.py", line 216, in <module>
    main()
  File "examples/train_model.py", line 130, in main
    world.parley()
  File "/home/presende/Work/ParlAI/parlai/core/worlds.py", line 601, in parley
    batch_act = self.batch_act(index, batch_observations[index])
  File "/home/presende/Work/ParlAI/parlai/core/worlds.py", line 574, in batch_act
    batch_actions = a.batch_act(batch_observation)
  File "/home/presende/Work/ParlAI/parlai/agents/seq2seq/seq2seq.py", line 446, in batch_act
    predictions, text_cand_inds = self.predict(xs, ys, cands)
  File "/home/presende/Work/ParlAI/parlai/agents/seq2seq/seq2seq.py", line 256, in predict
    token = self.v2t([preds.data[b][0]])
TypeError: 'int' object is not subscriptable
```

Just after
```
Dictionary: loading dictionary from /tmp/model_s2s.dict
[ num words =  25 ]
[creating task(s): babi:task1k:1]
[loading fbdialog data:/home/presende/Work/ParlAI/data/bAbI/tasks_1-20_v1-2/en-valid-nosf/qa1_train.txt]
[ training... ]
```

is printed in the terminal.

I'm using python3 and have all of the requirements installed. The system I'm trying to run this on does not have a GPU, and I have installed pytorch's no-CUDA version.

**Comment by avr248** [comment]
python3 + pytorch's CUDA version = same error!

**Comment by wolegechu** [comment]
It't from a PyTorch update. Fixed it in #280. You should modify three lines of the code.

**Comment by ptresende** [comment]
Thanks @wolegechu, I managed to get it working now, and will close the issue.
